packages/design-system-dashboard-cli/src/find-ds-components.js

```diff
@@ -56,17 +56,25 @@ function filterSearchedComponents(data, searchStrings = []) {
  *
  * @return {Object} - The component-name-keyed object containing the usage count
  */
-function tallyResults(vwComponents, contentBuildWC) {
+function tallyResults(vwComponents, contentBuildWC, uswdsComponents) {
   const data = {};
   /** @type {{[component: string]: string[]}} */
   const instancesByComponent = {};
+  // if data[componentName].uswds isn't present then initialize it, otherwise increment it
   function incrementCount(componentName, appName) {
     if (!data[componentName]) data[componentName] = { total: 0 };
     if (!data[componentName][appName]) data[componentName][appName] = 0;
     data[componentName][appName]++;
     data[componentName].total++;
   }
 
+  function incrementUswdsCount(componentName) {
+    if (!data[componentName].uswds) {
+      data[componentName].uswds = 0;
+    }
+    data[componentName].uswds++;
+  }
```

Comment:
Does the same thing as incrementCount but specifically for the uswds property only